### PR TITLE
DR documentation updates

### PR DESCRIPTION
**Pull request by RMcVelia**
## Context
DR documentation updates resulting from the recent DR test.

https://trello.com/c/M8apVWh5/2009-dr-test-documentation-updates

## Changes proposed in this pull request
Various modifications and additions to improve the process 

## Guidance to review
Read through changes and check they make sense

## Checklist

- [ ] I have performed a self-review of my code, including formatting and typos
- [ ] I have [cleaned the commit history](https://www.annashipman.co.uk/jfdi/good-pull-requests.html)
- [ ] I have added the `Devops` label
- [ ] I have attached the pull request to the trello card

### CODE REVIEW

**Comment by RMcVelia** [comment]
[DR test documentation updates](https://trello.com/c/M8apVWh5/2009-dr-test-documentation-updates)

File: documentation/disaster-recovery-testing.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Disaster Recovery testing
 
-This document covers the Disaster Recovery testing procedure for applications hosted on the Teacher Services AKS clusters based on scenerios details in the [Disaster recovery document](disaster-recovery.md).
+This document covers the Disaster Recovery testing procedure for applications hosted on the Teacher Services AKS clusters based on scenerios detailed in the [Disaster recovery document](disaster-recovery.md).
```
Comment (saliceti) [inline documentation/disaster-recovery-testing.md:3]: scenarios

File: documentation/disaster-recovery.md [hunk 2]
```diff
@@ -2,63 +2,99 @@
...
+### Freeze pipeline
+
+Alert developers that no one should merge to main.
+- In github setings, a user with repo admin privileges should update the *Branch protection rules* and set required PR approvers to 6
+
 ### Enable maintenance mode
 
 Run the *enable maintenance workflow* for the service and environment affected.
 
+If the maintenance page message needs to be updated at any time during the incident
```
Comment (saliceti) [inline documentation/disaster-recovery.md:34]: Can this be moved to https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/maintenance-page.md#update-content ?

File: documentation/disaster-recovery.md [hunk 3]
```diff
@@ -2,63 +2,99 @@
...
 
-The application's database is an Azure flexible postgres server. This provides a point-in-time restore (PTR) ability with the resolution of 1 second, available between 5min and 7days. PTR allows you to restore the source server to a point-in-time on a new copy of the server, not to the source server itself.
+The application database is an Azure flexible postgres server. This server has a point-in-time restore (PTR) ability with the resolution of 1 second, available between 5min and 7days. PTR allows you to restore the source server to a point-in-time on a new copy of the server. It does not update the source server itself in any way. Once the new server is available it can be accessed using [konduit.sh](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/scripts/konduit.sh) to check previous data, and data can then be recovered to the original server.
 
 The objectives are:
 - Create a separate new postgres database server
 - Restore data from the current live database to the new postgres database server from a particular point in time
 - Update data into the source database from the new PTR server
 
+### Start the incident process (if not already in progress)
```
Comment (saliceti) [inline documentation/disaster-recovery.md:74]: Can this be moved after "Stop the service"? We should shut down early

File: documentation/disaster-recovery-testing.md (hunk 1 above)
Comment (RMcVelia) [reply documentation/disaster-recovery-testing.md]: changed

File: documentation/disaster-recovery.md (hunk 2 above)
Comment (RMcVelia) [reply documentation/disaster-recovery.md]: moved and added link

File: documentation/disaster-recovery.md (hunk 3 above)
Comment (RMcVelia) [reply documentation/disaster-recovery.md]: moved as requested

**Review by saliceti — APPROVED** [review]

**Pull request merged**